binder/environment.yml

```diff
@@ -0,0 +1,24 @@
+name: calour-environment
+channels:
+  - conda-forge
+  - biocore
+dependencies:
+  - python
+  - nose
+  - pep8
+  - flake8=3.3.0
+  - numpy
+  - scipy
+  - pyqt>=5
+  - pandas
+  - matplotlib
+  - scikit-learn
+  - scikit-bio>=0.5.1
+  - biom-format
+  - statsmodels
+  - ipywidgets
+  - numpydoc
+  - sphinx
+  - sphinx_bootstrap_theme
+  - biom-format
+  - scikit-bio
```

Comment:
how does the binder dir work? just curious, could you point me to a tutorial/description?

Reply:
it creates a docker file for the repo installation and runs a new instance for each user
to specify the installation requirements, you supply conda/pip/batch files (like for travis)
see [here](https://mybinder.readthedocs.io/en/latest/using.html#environment-yml)